lib/ReactViews/Notification/shareConvertNotification.tsx

```diff
@@ -11,11 +11,11 @@ const Spacing: any = require("../../Styled/Spacing").Spacing;
 const RawButton: any = require("../../Styled/Button").RawButton;
 const TextSpan: any = require("../../Styled/Text").TextSpan;
 
-export const shareConvertNotification = (shareProps: any) => (
+export const shareConvertNotification = (messages: any) => (
```

Comment:
We have typings for this. Adding:
```suggestion
export const shareConvertNotification = (
  messages: NotUndefined<import("catalog-converter").ShareResult["messages"]>
) => (viewState: ViewState) => {
```
I realised that (according to typings at least) `messages` can be undefined. That should be handled in either Terria.ts or here.

Reply:
Looks like it's always an Array in catalog-converter code. We should change the types in catalog-converter.